### PR TITLE
Summay Statistics -Climatic Methods Menus.

### PR DESCRIPTION
**Pull request by aduut**
Created Summary Statistics Dialogue with all parameters

### CODE REVIEW

File: instat/dlgSummaryStatistic.Designer.vb [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    <System.Diagnostics.DebuggerStepThrough()> _
+    Private Sub InitializeComponent()
+        Me.ucrBase = New instat.ucrButtons()
+        Me.chkRemoveNA = New System.Windows.Forms.CheckBox()
+        Me.chkAppend = New System.Windows.Forms.CheckBox()
+        Me.chkConvert = New System.Windows.Forms.CheckBox()
+        Me.chkStatistics = New System.Windows.Forms.CheckBox()
+        Me.chkCounts = New System.Windows.Forms.CheckBox()
+        Me.chkPercents = New System.Windows.Forms.CheckBox()
+        Me.TextBox1 = New System.Windows.Forms.TextBox()
```
Comment (volloholic) [inline instat/dlgSummaryStatistic.Designer.vb]: What is this textbox? The code below is empty?

**Comment by volloholic** [comment]
Again  think this is and easy correction which could be made so that the code can be committed.

**Comment by stevekogo** [comment]
A new dialogue is needed for this.

File: instat/dlgSummaryStatistic.Designer.vb (hunk 1 above)
Comment (Lunalo) [inline instat/dlgSummaryStatistic.Designer.vb]: Finished fixing

**Pull request merged**